cmd/up.go

```diff
@@ -65,6 +65,14 @@ func UpCommand() cli.Command {
 			Name:  "custom-certs",
 			Usage: "Use custom certificates from a cert dir",
 		},
+		cli.StringFlag{
```

Comment:
We should be *very* consistent with rancher upgrade strategy. From what I recall, we've agreed on having maxUnavailable parameter in leu of batch interval and size

Reply:
yes, I'm yet to change it